includes/class-wp-job-manager-post-types.php

```diff
@@ -1473,6 +1473,11 @@ public static function get_job_listing_fields() {
 			$application_method_placeholder = __( 'https://', 'wp-job-manager' );
 		}
 
+		$job_expires_description = __( 'Job listing expires at the end of the selected day.', 'wp-job-manager' );
```

Comment:
I wonder if we could just say "... at the end of the day"?

Reply:
Yup, had that originally then I think I overthought it when in the interface. Changed.